gc/base/standard/ConfigurationStandard.cpp

```diff
@@ -98,14 +98,20 @@ MM_ConfigurationStandard::initialize(MM_EnvironmentBase* env)
 	return result;
 }
 
-
 void
 MM_ConfigurationStandard::initializeGCThreadCount(MM_EnvironmentBase* env)
 {
-
 	MM_Configuration::initializeGCThreadCount(env);
 
 #if defined(OMR_GC_CONCURRENT_SCAVENGER)
+	initializeConcurrentScavengerThreadCount(env);
```

Comment:
an existing problem, but still: this code (now a method) should really exist in ConfigurationGenerational (hence no need to even be virtual)

Reply:
`initializeGCThreadCount` and `initializeConcurrentScavengerThreadCount`
moved from `MM_ConfigurationStandard`  -> `MM_ConfigurationGenerational`